### ISSUE TITLE
Replace ppo:floral structure with po:reproductive shoot system

### ISSUE THREAD

**Issue by ramonawalls** [PlantPhenoOntology/ppo#87]
ppo:floral structure is defined as po:flower OR po:infloresence, but both of those are subclasses of po:reproductive shoot system. It might be better to just add an alternative label to reproductive shoot system. I don't remember the logic for creating a new union class.

**Comment by edenny** [comment]
Makes sense! I think we created floral structures because of counts in the NPN data. Observers are allowed to give estimated counts for either individual flowers or inflorescences depending on species (e.g. a birch catkin counts as 1 unit). But makes no difference if we use "repro shoot system" instead of "floral structure" I don't think.

**Comment by edenny** [comment]
Oh.... but we do have traits for "open flowers" (individual flowers), "open flower heads" (inflorescences), "open floral structures" (either flowers or inflorescences). We also have "pollen-releasing..." and "non-senesced.." of each of those. 

Does it work to instead have "open reproductive shoot system", "pollen-releasing reproductive shoot system" and "non-senesced reproductive shoot system"?

**Comment by ramonawalls** [comment]
@elhartley, you should also deprecate the ppo:foral structure and add an alternative label to po:reproductive shoot system. I think you can do that in the DOS-DP pattern file.

You should rename the classes for  "open floral structures" (either flowers or inflorescences), "pollen-releasing...", "unopened..." and "non-senesced..." as "open reproductive shoot system", "pollen-releasing reproductive shoot system", "unopened reproductive shoot system" and "non-senesced reproductive shoot system", make them subclasses of po:reproductive shoot system, and add alternative labels for each X state using "X floral structure".

**Comment by elhartley** [comment]
@ramonawalls The re-labeled term `senesced reproductive shoot system` (alt. label = `senesced floral structure`) currently has the following definition:

`A floral structure that has completed floral organ senescence.`

Should I leave this definition as is or modify it to:

`A reproductive shoot system that has completed floral organ senescence.`

or something else along this line?

**Comment by elhartley** [comment]
For consistency, should the `floral structure presence`, `floral structure absent`, and `floral structure present` classes also be renamed to `reproductive shoot system presence`, `reproductive shoot system absent`, and `reproductive shoot system present` with the original names added as alternative labels?

**Comment by ramonawalls** [comment]
Are the definitions not generated automatically (in which case it would change when you update the pattern file)? If not, we should look into creating an issue for definitions. Regardless, the definitions should use the primary label.

**Comment by elhartley** [comment]
These terms are being created from a ROBOT template ([PPO_plant_structures.tsv](https://github.com/PlantPhenoOntology/ppo/blob/87-replace-floral-structure/src/templates/PPO_plant_structures.tsv)), so the definitions aren't generated automatically. Of the 5 "x floral structure" terms that are being renamed, only the one I mentioned above has a definition (which I'll update to use the primary label).

Should I add definitions for the other 4 terms (`non-senesced reproductive shoot system`, `unopened reproductive shoot system`, `open reproductive shoot system`, and `pollen-releasing reproductive shoot system`) or leave that to be fixed as part of the to-be-created missing definition issue since there are also other terms that don't have definitions?

**Comment by ramonawalls** [comment]
@elhartley Yes, please add definitions to all the terms so that they are consistent with the primary label as you recommended in https://github.com/PlantPhenoOntology/ppo/issues/87#issuecomment-2571481076.

**Comment by elhartley** [comment]
Fixed by #107 and #109.
